src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Stream;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] empty = new String[]{};
+        StringBuilder readStrings = new StringBuilder();
+        File test = new File(fileName);
+
+        if (test.length() == 0) {
+            return empty;
```

Comment:
what do you do with this check?

Reply:
was checking if file is null but now, it is not necessary, doing the same check in the return